src/api/migrations/20190620192200-create-user.js

```diff
@@ -68,5 +68,5 @@ export default {
       type: Sequelize.DATE
     }
   }),
-  down: (queryInterface, Sequelize) => queryInterface.dropTable('Users')
+  down: (queryInterface, Sequelize) => queryInterface.dropTable('User')
```

Comment:
Please switch back to users

Reply:
@Ramadhan0 kindly implement @truestbyheart's feedback